client-web/src/app/components/header/header.component.html

```diff
@@ -33,6 +33,18 @@
                 <button mat-button (click)="hLayer2(menuChild.id)" *ngFor="let menuChild of serviceHLayer2"> {{menuChild.title}} </button>
                 
                 <!-- <mat-icon *ngIf="serviceHLayer3?.length">expand_more</mat-icon> -->
+
+
+            <div class="showHousehold">
```

Comment:
Why the class name is showHousehold.. Household is not something static. Its dynamic. Change classname to relevant name